### ISSUE TITLE
Don't hard-code first invoice number

### ISSUE THREAD

**Issue by franc6** [iccm-africa/iccm-backend#9]
The first invoice number is hard-coded as "2020001" with a comment of "The first invoice number of this year".  Instead, we can build the number dynamically as the 4-digit year followed by 001.

**Comment by ruliezz** [comment]
@franc6 I agree with this. Probably this was quick fix for the system to run in 2020. But indeed. this should be fixed. 
@AlbertJP any comments on this or ready to pick-up?
